### DISCUSSION TITLE
Multiple transports?

### DISCUSSION THREAD

**Discussion by YoshieraHuang** [libp2p/rust-libp2p#2534] [Q&A]
[libp2p docs](https://docs.libp2p.io/concepts/transport/#supporting-multiple-transports) says that the multiple transports is supported. Does rust-libp2p supprots this feature? If so, an example is appreciated.

**Comment by mxinden** [comment]
For others to follow along, answered in https://github.com/libp2p/rust-libp2p/issues/2538.
